frontend/integration-tests/tests/kubevirt/models/wizard.ts

```diff
@@ -76,11 +76,12 @@ export default class Wizard {
     await isLoaded();
   }
 
-  async addNIC(name: string, mac: string, networkDefinition: string) {
+  async addNIC(name: string, mac: string, networkDefinition: string, binding: string) {
```

Comment:
can we please have the same order of parameters for both `addNIC` and `addNic` methods? maybe also unite the names?

Reply:
Good point, that's probably my bad :(

Reply:
updated.